.github/component_owners.yml

```diff
@@ -36,6 +36,7 @@ components:
     - NathanielRN
     - Kausik-A
```

Comment:
Maybe we can now remove the component owners who are inactive?

Reply:
Done^. `Kausik-A` seems to be inactive from GitHub in general for a couple years too.